### DISCUSSION TITLE
Differences between user env var & sys env var

### DISCUSSION THREAD

**Discussion by Ace-Radom** [BartoszCichecki/LenovoLegionToolkit#1340] [General]
As I translated the new added resources, I noticed the description of `PATH` settings is:

https://github.com/BartoszCichecki/LenovoLegionToolkit/blob/9e8f7dc3bcf73400c0fc688789869f381d97b310/LenovoLegionToolkit.WPF/Resources/Resource.resx#L2174

But system environment variable and user environment variable are not the same, and I think they shouldn't be confused. Currently LLT writes `PATH` var of the current user, would it be better if we use `user's PATH environment variable` instead of `system ...`?

**Comment by BartoszCichecki** [comment]
Good suggestion. Just pushed the change.
